### PR TITLE
Mobile Navbar Adjustments

### PR DESCRIPTION
**Pull request by Jo-Sundaram**
### Updates

- Moved Hamburger menu to top left of the screen
- Changed background colour of navbar to white and made it wider
- Made navbar slightly transparent so you can still see what's on screen 

### Issues

- CSS transitions are not 100% smooth. Let me know if it's really noticeable or not, if it is I'll continue trying to fix it. 

mobile-navbar-gif

### CODE REVIEW

File: components/NavBar/ActiveLink.js [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+         </Link>
+        <div className = {`${router.pathname===href?styles.activePage : styles.inactivePage}`}>{children}</div>   
+       </div>
+    );
+
+
+}
+
+
+/* export default function ActiveLink({children,href}){
```
Comment (Monica-Vu) [inline components/NavBar/ActiveLink.js:25]: Remove any unnecessary comments please :)

File: components/NavBar/NavBar.js [hunk 2]
```diff
@@ -1,5 +1,63 @@
 import styles from "./NavBar.module.scss";
+import ActiveLink from "./ActiveLink";
+const leafLogo =  require("../../images/NavBar/leafLogo.svg");
+
+
+/* const NavBar=()=>
```
Comment (Monica-Vu) [inline components/NavBar/NavBar.js:6]: Remove any unnecessary comments please :)

File: components/NavBar/NavBar.module.scss [hunk 3]
```diff
@@ -1,18 +1,216 @@
 @import "../../theme/theme";
 
-.NavBar {
-    display: flex;
-    justify-content: space-around;
-    align-items: center;
+.NavBar {  // navbar container
+    // position: fixed;
```
Comment (Monica-Vu) [inline components/NavBar/NavBar.module.scss:4]: Remove any unnecessary comments please :)

File: components/NavBar/NavBar.module.scss [hunk 4]
```diff
@@ -1,18 +1,216 @@
...
   
     font-family: $secondary-header-font;
     font-size: $body-font-size;
-    color: $misty-pink;
+    color: $raisin-dark;
+    transition: width 300ms ease-in-out;
+  }
+
+
+// .technolgap{ // technolgap logo
```
Comment (Monica-Vu) [inline components/NavBar/NavBar.module.scss:24]: Remove any unnecessary comments please :)

File: components/NavBar/NavBar.module.scss [hunk 5]
```diff
@@ -1,18 +1,216 @@
...
+  } 
+  
+  .hamburgerContainer:hover ~.NavBar .navList{
+    left:auto;
+   
+  } 
+  
+  
+ 
+  // .hamburgerContainer:hover .bar1{
```
Comment (Monica-Vu) [inline components/NavBar/NavBar.module.scss:186]: Remove any unnecessary comments please :) (old CSS)

Comment (Jo-Sundaram) [reply components/NavBar/NavBar.module.scss]: comments removed!

**Review by Monica-Vu — APPROVED** [review]
@Jo-Sundaram MERGEEEEE

**Pull request merged**